domains/open.is-cool.dev

```diff
@@ -0,0 +1,18 @@
+{
+    "domain": "is-cool.dev",
+    "subdomain": "open",
+    "owner": {
+        "email": "epoxid@riseup.net"
+    },
+
+    "record": {
+        "A": [],
+        "AAAA": [],
+        "CNAME": "",
+        "MX": [],
+        "TXT": [],
+        "NS": ["ns1.dynv6.com","ns2.dynv6.com","ns3.dynv6.com"]
```

Comment:
Please provide a valid reason for requiring NS records.

Reply:
I am a university student from southern China. Apply for NS records for personal blogs and other personal learning projects

Reply:
Sorry, misread. It's not my request

Reply:
done